crates/rpc/rpc/src/eth/filter.rs

```diff
@@ -43,6 +49,7 @@ impl<Provider, Pool> EthFilter<Provider, Pool> {
         eth_cache: EthStateCache,
         max_logs_per_response: usize,
         task_spawner: Box<dyn TaskSpawner>,
+        stale_filter_ttl: Duration,
```

Comment:
we should add a note to the docs that this spawns a service task

Reply:
Good! 
I added a comment in the struct definition, should I add something else?